### DISCUSSION TITLE
Enabling Ethernet connectivity on a Tasmotized UbiBot MS1P.

### DISCUSSION THREAD

**Discussion by prakir77** [arendst/Tasmota#17764] [Q&A] [answered]
Could someone help me figure this out? Been trying various configurations and templates for almost half a month, and I still haven't figured this out. Would also like help on properly figuring out and configuring the motion detector settings for this device. Any help would be much appreciated, thanks!

**Comment by Jason2866** [comment, marked as answer]
Provide link to device and data sheets for. What have you tried?
No one can and will help you, without as much as possible Infos.

**Reply by barbudor** [reply]
You are the one with the device in hand and probably the first to tasmotized it
So you need to open it up and find the connections to find out the template

**Reply by prakir77** [reply]
Thank you both for the replies! I've looked, but couldn't find any information beyond the product page for the device, and it's retail listings on Amazon and other sellers.

This is the product: (https://ubibotus.com/ubibot-motion-sensor-ms1p/)

And it's internals: (https://1drv.ms/u/s!AiofHGmIywtgmmZmQbPEFLhQU-Zh?e=boJYe4)

The board appears to be an esp32 wroom 32

I tried using this to get an idea of what's what: (https://www.mischianti.org/wp-content/uploads/2021/02/ESP32-wroom-32-pinout-mischianti.jpg), but am unable to figure everything out on the particular device I have.

I tried the different POE device templates from Blakadder, including the one for the Olimex ESP32-POE Ethernet Development Board: (https://templates.blakadder.com/olimex_ESP32-POE.html), but had no success in enabling Ethernet connectivity on my device.

I tried flipping different GPIO pin configurations on the online Tasmota GUI for the Tasmotized Ubibot MS1P, alternating and reordering the Ethernet_MDC and Ethernet_MDIO across the 25+ individual GPIO pin configuration, but got nowhere. Sorry if I'm missing something obvious, very new to this whole thing. Any help would be much appreciated! Thank you very much in advance!

**Reply by barbudor** [reply]
This device use a WizNET W5500 embedded TCP/IP controller
It does not make use of the ESP32 Ethernet Controller but is a totally independant chip which handles everything

This is totally dumb from UbiBot because these chips are meant to be used with MCU that do not have an on-chip Ethernet controller like Arduino ATMega
With an ESP32, it is stupid as there is NO way to have it controlled by ESP's native TCP/IP stack.

Forget about getting Ethernet supportin Tasmota with that

**Reply by prakir77** [reply]
Thanks for the reply, really appreciate it. That's unfortunate, was really hoping to get this working -especially since this device does support this odd POE configuration via its original firmware. Thanks anyway.
